bedrock/firefox/templates/firefox/challenge-the-default/landing-base.html

```diff
@@ -261,56 +242,56 @@ <h2>{{ compare_title|safe }}</h2>
               <th scope="row">{{ compare_passwords }}</th>
-              <td class="firefox" aria-label="checkmark">&#x2713;</td>
-              <td class="safari" aria-label="checkmark">&#x2713;</td>
-              <td class="chrome" aria-label="checkmark">&#x2713;</td>
-              <td class="edge" aria-label="checkmark">&#x2713;</td>
+              <td class="firefox" aria-label="checkmark">✓</td>
+              <td class="safari" aria-label="checkmark">✓</td>
+              <td class="chrome" aria-label="checkmark">✓</td>
+              <td class="edge" aria-label="checkmark">✓</td>
             </tr>
             <tr>
               <th scope="row" class="break-word">{{ compare_search }}</th>
-              <td class="firefox" aria-label="checkmark">&#x2713;</td>
-              <td class="safari" aria-label="checkmark">&#x2713;</td>
-              <td class="chrome" aria-label="checkmark">&#x2713;</td>
-              <td class="edge" aria-label="checkmark">&#x2713;</td>
+              <td class="firefox" aria-label="checkmark">✓</td>
+              <td class="safari" aria-label="checkmark">✓</td>
+              <td class="chrome" aria-label="checkmark">✓</td>
+              <td class="edge" aria-label="checkmark">✓</td>
             </tr>
             <tr>
               <th scope="row">{{ compare_sync }}</th>
-              <td class="firefox" aria-label="checkmark">&#x2713;</td>
-              <td class="safari" aria-label="checkmark">&#x2713;</td>
-              <td class="chrome" aria-label="checkmark">&#x2713;</td>
-              <td class="edge" aria-label="checkmark">&#x2713;</td>
+              <td class="firefox" aria-label="checkmark">✓</td>
+              <td class="safari" aria-label="checkmark">✓</td>
+              <td class="chrome" aria-label="checkmark">✓</td>
+              <td class="edge" aria-label="checkmark">✓</td>
             </tr>
             <tr>
               <th scope="row">{{ compare_block }}</th>
-              <td class="firefox" aria-label="checkmark">&#x2713;</td>
-              <td class="safari" aria-label="checkmark">&#x2713;</td>
+              <td class="firefox" aria-label="checkmark">✓</td>
+              <td class="safari" aria-label="checkmark">✓</td>
               <td class="chrome"></td>
-              <td class="edge" aria-label="checkmark">&#x2713;</td>
+              <td class="edge" aria-label="checkmark">✓</td>
             </tr>
             <tr>
               <th scope="row">{{ compare_profit }}</th>
-              <td class="firefox" aria-label="checkmark">&#x2713;</td>
+              <td class="firefox" aria-label="checkmark">✓</td>
               <td class="safari"></td>
               <td class="chrome"></td>
               <td class="edge"></td>
             </tr>
             <tr>
               <th scope="row">{{ compare_privacy }}</th>
-              <td class="firefox" aria-label="checkmark">&#x2713;</td>
+              <td class="firefox" aria-label="checkmark">✓</td>
```

Comment:
Q: would it be worth to fix #13357 by replacing these with some more plain ASCII like `•` bullet?

Reply:
I think this is enough of an edge case that we should keep the more readable ✓

Reply:
The re-training of the models is in progress (I'm updating the tracking bugs and epics in the issue above), but those og languages as _de_ & _fr_ are expensive so they plan to do some optimizations first — but yea, it's a known issue elsewhere and they have these characters break in more important places already, so that's ought to be fixed by distributing new models once they are available…